### ISSUE TITLE
Download worker: Ensure removal of files after use

### ISSUE THREAD

**Issue by JakobMiksch** [klips-project/rabbitmq-worker#40]
The download worker currently only downloads files, but they should be removed at one point, once they are not needed anymore.

**Comment by weskamm** [comment]
not sure if this is the right job for the download worker or a separate cleanup worker.
On the other hand, this may be part of a bigger generic `ROLLBACK` implementation every worker should implement for error cases... Needs discussion i think

**Comment by JakobMiksch** [comment]
Yes, true. This might be a more generic problem, but I identified it initially with the download worker.

**Comment by JakobMiksch** [comment]
I updated the description of the issue.
